Resources/Prototypes/_RMC14/Entities/Clothing/Masks/masks.yml

```diff
@@ -68,6 +70,11 @@
     coverage: MOUTH
   - type: Item
     storedRotation: -90
+  - type: HideLayerClothing # TODO port pulled down sprite from SS13
+    slots:
+    - Hair
+    - HeadTop
+    - HeadSide
```

Comment:
Drop these changes as they're not related to the pr
(it needs to be changed to support optional toggle)